Makefile

```diff
@@ -349,7 +353,7 @@ format-packer: ## Reformat all Packer HCLs
 	packer fmt -recursive packer/
 
 .PHONY: format
-format: format-python format-prettier format-rust format-packer ## Reformat all code
+format: format-python format-shell format-prettier format-rust format-packer ## Reformat all code
```

Comment:
This is not necessarily something to fix in this PR, but more of an observation. We have all these different formatting targets for the various kinds of files in our repository (python, shell, rust, packer), and then we have `format-prettier`, which isn't a file type, but a tool, and it does the formatting for all the JS, YAML, Markdown, and TOML in one go, with no ability to target one specifically. This is a bit of an weird incongruity. It's also a bit of a downer that if I just want to make sure my TOML file is formatted properly, I have to sit through formatting all the JS, YAML, and Markdown files, just because the TOML formatting currently happens to be at the end of that script.

Reply:
So on one hand I agree, but timing it - it's so quick that I'm not sure it's worth the added complexity of chopping it up.

real    0m13.621s
user    0m0.793s
sys     0m0.360s